src-test/org/etools/j1939_84/bus/j1939/packets/SupportedSPNTest.java

```diff
@@ -27,6 +38,12 @@ public void testError() {
         assertEquals(expected, instance.toString());
     }
 
+    @Test
+    public void testHashCode() {
+        SupportedSPN instance = new SupportedSPN(new int[] { 0x01, 0x02, 0x1D, 8 });
```

Comment:
This test should make two instances and verify that the hashCode of both is the same.  When two objects are equal, their hashCode must be the same (per the object contract).  (It’s possible that two instances are not equal but the hashCodes just happen to be the same, though that’s rare)